Explain why piracy sites can't be included (#41)
yt-dlp/yt-dlp-wiki: FAQ.md

     - Or, if you'd rather use a cookies file instead of having yt-dlp extract cookies from your browser, you can use the `--cookies` option, e.g. `--cookies cookies.txt`. Note that the cookies need to be exported from a fresh browser session (see step 1) *within the past 30 minutes*. [More info on exporting cookies](#how-do-i-pass-cookies-to-yt-dlp)

NOTE: This method reportedly works best with Firefox, and some users have reported problems getting it to work with Edge.

### Why is there a rule against websites "primarily used for piracy"?

The short answer is that supporting piracy sites would in all likelihood result in the project being shut down.
In the past, other similar projects have been shut down due to supporting piracy sites.

> The yt-dlp maintainers are of the position that supporting these sites would be of no benefit to the project, but rather that doing so would only put the project's continued existence at risk.

Reviewer: ```suggestion
The yt-dlp maintainers are of the position that supporting such sites is not worth the risk.
```

     - Or, if you'd rather use a cookies file instead of having yt-dlp extract cookies from your browser, you can use the `--cookies` option, e.g. `--cookies cookies.txt`. Note that the cookies need to be exported from a fresh browser session (see step 1) *within the past 30 minutes*. [More info on exporting cookies](#how-do-i-pass-cookies-to-yt-dlp)

NOTE: This method reportedly works best with Firefox, and some users have reported problems getting it to work with Edge.

### Why is there a rule against websites "primarily used for piracy"?

> The short answer is that supporting piracy sites could both get the project shut down and the maintainers into legal trouble.

Reviewer: keep it simple IMO
```suggestion
The short answer is that supporting piracy sites would in all likelihood result in the project being shut down.
```

In the past, other similar projects have been shut down due to supporting piracy sites.

> If you are willing to take the risk of fighting (and potentially losing) a year-long legal battle, you are free to fork the project and make your own rules.
> The current maintainers of yt-dlp have decided that they are not willing to take that risk.

Reviewer: I would consolidate these two lines and maybe omit the bit about forking
```suggestion
The yt-dlp maintainers are of the position that supporting these sites would be of no benefit to the project, but rather that doing so would only put the project's continued existence at risk.
```

### What does "sites primarily used for piracy" mean?

There is no clear-cut definition here, as any site that hosts user-generated content can be used for piracy.
However, common grounds for exclusion from yt-dlp are:

> - It is difficult to find a single non pirated video

Reviewer 2: ```suggestion
- It is difficult to find a single non-pirated video
```

Reviewer: ```suggestion
- It is difficult to find a single non-pirated video on the site
```

- The site is being discussed as a safe haven for pirated content in online forums
- The site's operators are completely anonymous or unknown

Any of these reasons are enough for a site to be excluded from yt-dlp.

> This list is not exhaustive. It is up to the discretion of the maintainers if a site will be included in yt-dlp or not.

Reviewer: ```suggestion
This list is not exhaustive. It is up to the discretion of the maintainers as to whether or not a site will be included in yt-dlp.
```
